modules/ROOT/pages/publishing-a-rest-api.adoc

```diff
@@ -0,0 +1,16 @@
+= Publishing a REST API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule provides a dedicated toolkit to facilitate the development and exposure of REST APIs. APIkit Rest is a simple framework that helps you with API-first development, enforcing good API implementation practices. You can add this toolkit to your instance of Anypoint Studio. +
+
+Rather than spending weeks or months building an API, you can use this toolkit to develop, document, and test your API within a few days or even hours.
```

Comment:
Could be replaced by "quickly" above.

Note also that we say the kit helps "you" do a variety of things: develop, expose, document, test.... Be sure to standardize to one set of attributes, rather than adding or subtracting user tasks. You might standardize to a list of only the main user goals, if there are too many.

Another way to say this, if we continue with this topic, is "Using APIkit, you can reduce your API development time from weeks or months to days or hours."